lang/nodejs14/Portfile

```diff
@@ -50,14 +46,18 @@ checksums               rmd160  fb95405a2acaee5064a021e2dc8a6a6493964360 \
 
 distname                node-v${version}
 
-depends_build-append    port:pkgconfig
-
 set py_ver              3.10
 set py_ver_nodot        [string map {. {}} ${py_ver}]
 
+depends_build-append    port:pkgconfig
+
 depends_lib-append      path:lib/pkgconfig/icu-uc.pc:icu \
-                        port:python${py_ver_nodot} \
-                        port:zlib
+                        path:lib/libz.dylib:zlib \
```

Comment:
Unless something changed while I wasn't looking, the zlib port is the only port that installs lib/libz.dylib, so there's no reason to change `port:zlib` to `path:lib/libz.dylib:zlib`.